packages/debugger/src/factory.ts

```diff
@@ -34,10 +33,7 @@ export class ReadOnlyEditorFactory {
     const mimeTypeService = this._services.mimeTypeService;
     const editor = new CodeEditorWrapper({
       model: new CodeEditor.Model({
-        sharedModel: createStandaloneCell({
-          cell_type: 'code',
-          source: content
-        }),
+        sharedModel: YFile.create(content),
```

Comment:
> In this use case, when disposing the CodeEditor.Model we are not disposing the sharedModel because the CodeEditor.Model leaves that responsibility to its child (The class that extends it, DocumentModel or CellModel).

> I think we should do the same trick we do in the CellModel:
this._standaloneModel = typeof options.sharedModel === 'undefined';

Maybe that trick is not a good idea, since in this case we will not dispose the sharedModel.

Reply:
Good point, I will set the source after creating the model.